sdk/ml/azure-ai-ml/tests/internal_utils/unittests/test_get_content_hash.py

```diff
@@ -28,19 +28,18 @@ def test_get_content_hash_should_not_change(self, tmp_path: Path):
             with open(full_file_path, "wb") as f:
                 f.write(test_file_contents)
 
+        hash = get_content_hash(tmp_path)
         actual_hash = hashlib.sha256()
         actual_hash.update(b"4")
-        actual_hash.update(b"#test_get_content_hash_should_n0/file1.txt#" + str(len(content1)).encode())
-        actual_hash.update(b"#test_get_content_hash_should_n0/folder1/file2.txt#" + str(len(content2)).encode())
-        actual_hash.update(b"#test_get_content_hash_should_n0/Folder2/file3.txt#" + str(len(content3)).encode())
-        actual_hash.update(b"#test_get_content_hash_should_n0/Folder2/folder1/file4.txt#" + str(len(content4)).encode())
+        actual_hash.update(b"#file1.txt#" + str(len(content1)).encode())
+        actual_hash.update(b"#folder1/file2.txt#" + str(len(content2)).encode())
+        actual_hash.update(b"#Folder2/file3.txt#" + str(len(content3)).encode())
+        actual_hash.update(b"#Folder2/folder1/file4.txt#" + str(len(content4)).encode())
         actual_hash.update(content1)
         actual_hash.update(content2)
         actual_hash.update(content3)
         actual_hash.update(content4)
-
-        assert (actual_hash.hexdigest() == get_content_hash(tmp_path)
-                == "4e375359128cd904a93a8b7c2cc883f2ae8ca40c7870136f258be5ef78f7e537")
+        assert actual_hash.hexdigest() == hash == "3f99429630ebd5882337eedef79dc029a9b406338cd6f466206aef2c951453be"
```

Comment:
may use a variable name like below and add comment to make sure people won't change this in future and break this
expected_hash_do_not_change = 'xxx'

Reply:
updated